src/main/webapp/app/core/course/shared/course-sidebar/course-sidebar.component.html

```diff
@@ -135,7 +135,7 @@
                 </li>
             }
 
-            @if (isNavbarCollapsed() || !sidebarItemsBottom().length) {
+            @if ((isNavbarCollapsed() || !sidebarItemsBottom().length) && canExpand()) {
```

Comment:
This condition is quite complex, a computed signal with a proper name would be better